src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -270,7 +287,13 @@ ReplicateActionResponse compute(String chainTaskId) {
             return ReplicateActionResponse.failureWithStdout(cause,
                     postResponse.getStdout());
         }
-        return ReplicateActionResponse.successWithStdout(appResponse.getStdout());
+        return ReplicateActionResponse.successWithLogs(
+                new ComputeLogs(
```

Comment:
Builder everywhere or constructor everywhere for consistency? (L272, L291)

Reply:
Sure, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/448/commits/8c658b4a32f4983869b8b65194d62f122bfa9ded.